packages/react-components/src/select/stories/transparent.stories.mdx

```diff
@@ -0,0 +1,233 @@
+import { HocWarning, InstallReactComponents, Props, ReactComponentsLayout, WarningMessage } from "@blocks";
+import { MagnifierIcon } from "@react-components/icons";
+import { Meta, Preview, Story } from "@storybook/addon-docs/blocks";
+import { PureSelect, Select, selectItem } from "@react-components/select";
+import { RefLogger, createComponentSection, paramsBuilder } from "@utils";
+import Imports from "./imports.mdx";
+
+<Meta
+    title={createComponentSection("Select/transparent")}
+    parameters={paramsBuilder()
+        .sortPriority(4)
+        .canvasLayout({ width: "80%" })
+        .docsLayout(<ReactComponentsLayout semanticPath="/modules/dropdown" />)
+        .build()}
+/>
+
+# Usage
+
+<WarningMessage>
+    This component have the same feature set as the <a href="?path=/docs/components-select-standard--default-story">standard select</a> component.
+</WarningMessage>
+
+# Transparent Select
+
+A select can be transparent in order to be more discrete.
+
+## Installation
+
+<InstallReactComponents imports={<Imports />} />
+
+## Props
+
+<HocWarning componentName="Dropdown" semanticPath="/modules/dropdown" />
+
+<Props
+    componentsDefinitions={[
+        { displayName: "Select", component: PureSelect },
+        { displayName: "Select.Item", component: PureSelect.Item }
+    ]}
+/>
+
+## Usage
+
+<WarningMessage>
+    This component have the same feature set as the <a href="?path=/docs/components-select-standard--default-story">standard select</a> component.
+</WarningMessage>
+
+### Constants
+
+The following constants are used in the examples below:
+
+export const GALAXIES = [
+    selectItem("Andromeda", "Andromeda"),
+    selectItem("Antennae", "Antennae"),
+    selectItem("Backward", "Backward"),
+    selectItem("Black Eye", "Black Eye"),
+    selectItem("Butterfly", "Butterfly"),
+    selectItem("Cartwheel", "Cartwheel"),
+    selectItem("Cigar", "Cigar"),
+    selectItem("Circinus", "Circinus"),
+    selectItem("Coma Pinwheel", "Coma Pinwheel"),
+    selectItem("Comet", "Comet"),
+    selectItem("Cosmos Redshift 7", "Cosmos Redshift 7"),
+    selectItem("Eye of Sauron", "Eye of Sauron"),
+    selectItem("Fireworks", "Fireworks"),
+    selectItem("Hockey Stick", "Hockey Stick"),
+    selectItem("Hoag's Object", "Hoag's Object"),
+    selectItem("Large Magellanic Cloud", "Large Magellanic Cloud"),
+    selectItem("Lindsay-Shapley Ring", "Lindsay-Shapley Ring"),
+    selectItem("Little Sombrero", "Little Sombrero"),
+    selectItem("Medusa Merger", "Medusa Merger"),
+    selectItem("Mice", "Mice"),
+    selectItem("Milky Way", "Milky Way"),
+    selectItem("Needle", "Needle"),
+    selectItem("Southern Pinwheel", "Southern Pinwheel"),
+    selectItem("Sunflower", "Sunflower"),
+    selectItem("Tadpole", "Tadpole"),
+    selectItem("Whirlpool", "Whirlpool")
+];
```

Comment:
You should use the same data set as the standard select. This one is pretty big, I used it for the search and multiple select because they needed a big dataset in order to showcase all the features. Don't need this for a transparent select.